docs/content/en/templates/sitemap-template.md

```diff
@@ -97,6 +100,7 @@ Defaults for `<changefreq>`, `<priority>` and `filename` values can be set in th
   changefreq = "monthly"
   priority = 0.5
   filename = "sitemap.xml"
+  exclude = true
```

Comment:
You should also change the paragraph above.

Reply:
Can you be more specific? I'm not following.

Reply:
Never mind I figured it out. "exclude" wasn't mentioned in the doc's note that it can be set in the main config. Fixed.